content/en/docs/overview/variables.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Variables"
+weight: 210
+---
+
+The short assignment statement `:=` declares a variable and assigns a value to it.
+The type of variable is inferred from the value (type inference).
+```golang
+package main
+
+import "fmt"
+
+func main() {
+	name := "foo.txt"
+	size := 42
+	isFile := true
+
+	fmt.Println(name, size, isFile)
+}
+```
+
+Once a variable is declared you can assign values to it with the regular assignment `=`.
+```golang
+package main
+
+import "fmt"
+
+func main() {
+	name := "foo.txt"
+
+	name = "bar.csv"
+
+	fmt.Println(name)
+}
+```
+
+The `var` keyword decalres a variable without assigning a value to it.
```

Comment:
Maybe a short sentence that we need to explicitly define the type if we don't assign a value?  
```suggestion
The `var` keyword declares a variable without assigning a value to it.
```